redis/conn.go

```diff
@@ -78,6 +79,7 @@ type dialOptions struct {
 	writeTimeout time.Duration
 	dialer       *net.Dialer
 	dial         func(network, addr string) (net.Conn, error)
```

Comment:
As this is internal we could always use dialContext with a context.Background if need.

Reply:
Fix!